typescript/src/bitcoin.ts

```diff
@@ -525,7 +523,12 @@ export function createKeyRing(
  * @returns Hash as a 20-byte un-prefixed hex string.
  */
 export function computeHash160(text: string): string {
-  return hash160.digest(Buffer.from(text, "hex")).toString("hex")
+  const sha256Hash = utils.sha256(
+    Hex.from(Buffer.from(text, "hex")).toPrefixedString()
```

Comment:
I'd be very careful here as we pass the `0x`-prefixed value to the `utils.sha256` hash function. I checked Ethers docs and looks like the prefix is not taken into account while calculating hashes but, just in case, can we cover this function with some basic unit tests to ensure it always works as expected? 

Same for `computeHash256` and `computeScriptHash`.